src/presentation/services/ingest-event-server.ts

```diff
@@ -0,0 +1,90 @@
+import { ClientConfiguration } from '../value-objects/client-configuration'
+import { Logger } from '../../logger/logger'
+import { IngestEventObserver } from '../interfaces/ingest-event-observer'
+import { IngestEvent } from '../value-objects/ingest-event'
+import { ClientConnectionServer } from '../interfaces/client-connection-server'
+import { QueuePoolObserver } from '../emitters/queue-pool-observer'
+
+export class IngestEventServer {
+  private readonly queueSubscriptions: Map<string, Set<string>> = new Map()
+  private readonly logger: Logger
+
+  public constructor(
+    private readonly clientConnectionServer: ClientConnectionServer,
+    private readonly ingestEventObserver: IngestEventObserver,
+    private readonly queuePoolObserver: QueuePoolObserver,
+    logger: Logger,
```

Comment:
Aren't you missing an accessibility modifier here? :)
```suggestion
    private readonly logger: Logger,
```

Reply:
The class member `logger` is defined above the constructor as `private readonly`. The reason it is done this way, is so that we can create a logging context for the logger, where the class name is added as metadata (the `logger.tag(this.constructor.name)` part).